std/math.d

```diff
@@ -6382,6 +6382,27 @@ if (isFloatingPoint!(X))
  */
 bool isIdentical(real x, real y) @trusted pure nothrow @nogc
 {
+    if (__ctfe)
+    {
+        if (x !is y) return false;
+        if (x == x) return true;
```

Comment:
Shouldn't that be `x == y`?

Reply:
Yes

Reply:
Ah, now I remember. It doesn't matter: the only circumstance where `x is y` but not `x == y` is if both x and y are NaN. So checking `x == x` is the same as checking `x == y` (it's a NaN check).